### PR TITLE
Bucket Explorer

### LINKED ISSUE #351: Bucket Explorer
Create a general purpose Bucket Explorer Tool that will work with any Bucket backend, and then remove Minio Console

### PR DESCRIPTION
**Pull request by jmthomas**
closes #351

### CODE REVIEW

**Comment by jmthomas** [comment]
Still need to add tests and verify with GCP but pretty close.

File: compose.yaml [hunk 1]
```diff
@@ -30,8 +30,8 @@ services:
   openc3-minio:
     image: "${OPENC3_REGISTRY}/openc3inc/openc3-minio:${OPENC3_TAG}"
     # Uncomment to run unit tests against the minio server
-    # ports:
-    #   - "127.0.0.1:9000:9000"
+    ports:
+      - "127.0.0.1:9000:9000"
```
Comment (ryanmelt) [inline compose.yaml:34]: Needs to be commented back out.

File: openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-bucketexplorer/.browserslistrc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+edge >= 80
+# Firefox ESR is extended support release
+firefox ESR
+# Safari 13.1 released Mar 24, 2020
+# See https://en.wikipedia.org/wiki/Safari_version_history
+safari >= 13.1
+not dead
+not ie 11
+# Opera mini doesn't have built-in support for ES6
+not op_mini all
```
Comment (ryanmelt) [inline openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-bucketexplorer/.browserslistrc:16]: missing newline at end of file.

File: openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-bucketexplorer/package.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build --no-module",
+    "test:unit": "vue-cli-service test:unit",
+    "lint": "vue-cli-service lint",
+    "serve:standalone": "vue-cli-service serve --mode standalone",
+    "test:components": "vue-cli-service test:components"
+  },
+  "resolutions": {
+    "acorn": "npm:acorn-with-stage3",
```
Comment (ryanmelt) [inline openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-bucketexplorer/package.json:14]: This line cost me three hours.  It keeps tools from building outside of the monorepo for some reason.  Do you know if we need these resolutions anymore?

**Review by ryanmelt — CHANGES_REQUESTED** [review]
I need to try it, but looks very well done.

File: openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-bucketexplorer/package.json (hunk 3 above)
Comment (jmthomas) [reply openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-bucketexplorer/package.json:14]: I don't know why that is there. I just build with our scripts.

**Comment by ryanmelt** [comment]
Also, I think we can remove minio console once this is complete.

**Comment by jmthomas** [comment]
> Also, I think we can remove minio console once this is complete.

Agreed ... eventually. I think we should leave minio console for now to avoid disrupting exiting users and mark it as deprecated.

**Comment by jmthomas** [comment]
Tested locally, AWS, and GCP

**Pull request merged**